NewForce-Cohort-9/bad-blood-brotherhood
# Rev branch2

## What Changed

1.  Completed basic layout for index.html and added CSS.
2. Added Hyperlinks for other HTML page.

## How to Test – *** rev-branch2 ***
Switch to my branch in terminal:
git fetch --all
git checkout rev-branch2
Check index.html in server and approve CSS.
Send me any errors you find, and I will correct.
If it looks correct, give me a thumbs up for confirmation. Thanks!
